plugins/modules/k8s_exec.py

```diff
@@ -212,7 +212,8 @@ def execute_module(module, k8s_ansible_mixin):
         rc = int(err["details"]["causes"][0]["message"])
 
     module.deprecate(
-        "The 'return_code' return key is deprecated. Please use 'rc' instead.",
+        "The 'return_code' return key is deprecated. Please use 'rc' instead. "
```

Comment:
```suggestion
        "The 'return_code' return key is being renamed to 'rc'. "
```
I think the fact that we specifically tell the user to do something that they are already doing is partly why this is confusing.